plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/controller/Content.js

```diff
@@ -104,8 +106,10 @@ Ext.define('NX.controller.Content', {
     // install new feature view
     content.add(cmp);
 
-    // fire activate event to view component
-    cmp.fireEvent('activate', cmp);
```

Comment:
@djsauble the purpose is to allow for components to do something after it has been visually activated:

http://docs.sencha.com/extjs/4.2.3/#!/api/Ext.AbstractComponent-event-activate

ie. the view is now visible, give it a chance to do something.

There are a handful of 'activate' event handlers, though I've not looked to see if they are specific to the feature view here or not.